src/app/models/user/user.go

```diff
@@ -1,10 +1,10 @@
 package user
 
 import (
+	"config/db"
 	"crypto/rand"
 	"errors"
 	r "github.com/dancannon/gorethink"
```

Comment:
No, it imports the package with 'r' as its name.

On Thu, Jul 2, 2015, 12:57 PM Nate Thompson <notifications@github.com>
wrote:

> In src/app/models/user/user.go
> <https://github.com/EaseApp/web-backend/pull/10#discussion_r33816440>:
>
> > @@ -1,10 +1,10 @@
> >  package user
> >
> >  import (
> > +	"config/db"
> >  	"crypto/rand"
> >  	"errors"
> >  	r "github.com/dancannon/gorethink"
>
> Is this "r" a typo?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/EaseApp/web-backend/pull/10/files#r33816440>.
>